code/controllers/subsystem/ticker.dm

```diff
@@ -143,6 +143,11 @@ SUBSYSTEM_DEF(ticker)
 	return ..()
 
 /datum/controller/subsystem/ticker/fire()
+	if(world.time > 30 MINUTES && !GLOB.cyropods_enabled)
```

Comment:
tEchNicAlLy world.time is from bootup so people will be able to cryo before 30 minutes round-time has passed

Reply:
Do we need another vote to determine if this 3 minutes is too big of a time lag?

Reply:
Council do not care